src/main/java/net/minecraftforge/fmp/item/ItemMultipart.java

```diff
@@ -0,0 +1,83 @@
+package net.minecraftforge.fmp.item;
+
+import net.minecraft.block.SoundType;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.EnumActionResult;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.EnumHand;
+import net.minecraft.util.SoundCategory;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Vec3d;
+import net.minecraft.world.World;
+import net.minecraftforge.fmp.multipart.IMultipart;
+import net.minecraftforge.fmp.multipart.MultipartHelper;
+
+/**
+ * {@link ItemMultipart} is an {@link Item} that can handle multipart placement.<br/>
+ * Implement {@link ItemMultipart#place(World, BlockPos, EnumFacing, Vec3, ItemStack, EntityPlayer)} and optionally
+ * override {@link ItemMultipart#consumeItem(ItemStack)} and {@link ItemMultipart#getPlacementSound(ItemStack)} to place
+ * your part in the world.
+ */
+public abstract class ItemMultipart extends Item implements IItemMultipartFactory
```

Comment:
It is essentially a default implementation of an item which places a multipart, much like ItemBlock is a default implementation of an item which places a block - however, it is not necessary and one could place the multipart themselves in onItemUse.